lib/documents.js

```diff
@@ -164,6 +168,56 @@ export async function register({
   }
 }
 
+/**
+ * Encrypts the document either once (with the given recipients) or recursively
+ * in a chain (using the recipientChain).
+ *
+ * @param {object} options - Options hashmap.
+ * @param {object} options.document - The document to register.
+ * @param {Array<string>} recipients - See `register()` docstring.
+ * @param {Array<Array<string>>} recipientChain - See `register()` docstring.
+ *
+ * @returns {Promise<object>} Resolves with the JWE, the result of encryption.
+ */
+export async function encrypt({document, recipients, recipientChain}) {
+  if(!(recipients || recipientChain)) {
+    throw new BedrockError(
+      'Encryption recipients or recipientChain is required.',
+      'ValidationError', {
+        public: true,
+        httpStatusCode: 400
+      });
+  }
+
+  const obj = document;
+  const keyResolver = createKeyResolver();
+  const cipher = new Cipher();
+
+  if(recipientChain) {
+    if(!Array.isArray(recipientChain)) {
+      throw new BedrockError(
+        '"recipientChain" must be an array.',
+        'ValidationError', {
+          public: true,
+          httpStatusCode: 400
+        });
```

Comment:
I recommend we move this up to the top of the main function so we don't need to do validation in the helper.

Reply:
Moved.